packages/@aws-cdk/aws-signer/README.md

```diff
@@ -9,12 +9,40 @@
 >
 > [CFN Resources]: https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib
 
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development.
+> They are subject to non-backward compatible changes or removal in any future version. These are
+> not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be
+> announced in the release notes. This means that while you may use them, you may need to update
+> your source code when upgrading to a newer version of this package.
+
 ---
 
 <!--END STABILITY BANNER-->
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+Define a Signer SigningProfile:
 
 ```ts
-import signer = require('@aws-cdk/aws-signer');
+import * as signer from '@aws-cdk/aws-signer';
+
+const signingProfile = new signer.SigningProfile(this, 'SigningProfile', { 
+  platformId: 'xxxxxx' 
+} );
 ```
+
+> **Note**: To get the list of available platforms, you can run aws-cli command of `aws signer list-signing-platforms`
+
+Define a Signer SigningProfile with validity period:
+
+Specifies the duration in the period that the signing profile is valid.
+
+```ts
+import * as cdk from '@aws-cdk/aws-core';
+
+const signingProfile = new signer.SigningProfile(this, 'SignginProfile', {
+  platformId: 'xxxxxx',
+  signatureValidityPeriod: cdk.Duration.days(365), // Default to 135 months
+}) 
+```
```

Comment:
Something simple like this would be sufficient.

```suggestion
A signing profile is valid by default for 135 months. This can be modified by specifying the `signatureValidityPeriod` property.
```
```